Libraries/Network/RCTHTTPRequestHandler.m

```diff
@@ -11,9 +11,6 @@
 
 #import "RCTAssert.h"
 #import "RCTConvert.h"
-#import "RCTEventDispatcher.h"
-#import "RCTImageLoader.h"
-#import "RCTLog.h"
 #import "RCTUtils.h"
```

Comment:
@tadeuzagallo - you are absolutely correct! `pod lib lint` ran successfully with all of those removed :+1: